saleor/graphql/core/utils/__init__.py

```diff
@@ -45,6 +49,45 @@ def validate_image_file(file, field_name):
         )
 
 
+def validate_video_url(url: str, field_name: str) -> Tuple[str, str]:
```

Comment:
Maybe this function should be moved to `salor/core/utils` instead, it's not GraphQL-specific. Also, we could add a file `media.py` in this utils module and move the function there.